src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/util/ManagedIndexUtils.kt

```diff
@@ -398,6 +398,10 @@ val ManagedIndexMetaData.isFailed: Boolean
         return false
     }
 
+// Adding predicate extension to allow cleaner checks since policyCompleted is nullable
+val ManagedIndexMetaData.isPolicyCompleted: Boolean
```

Comment:
Can we just make policyCompleted non-nullable? We're parsing the value in ManagedIndexMetaData and then constructing the data class, we could just say something like `policyCompleted = policyCompleted ?: false` to make sure the value on the ManagedIndexMetaData is actually a boolean. And then where we check `if (policyCompleted != null) ...` we can just check if its true then show it.

@jinsoor-amzn Thoughts on that too?

Reply:
I agree with making ```policyCompleted``` non-nullable but given the scope of ```ManagedIndexMetaData```'s usage both in our logic and tests, let's do this change in a separate PR.